pkg/cluster/k8sres.go

```diff
@@ -944,7 +961,7 @@ func (c *Cluster) getPodEnvironmentConfigMapVariables() ([]v1.EnvVar, error) {
 	configMapPodEnvVarsList := make([]v1.EnvVar, 0)
 
 	if c.OpConfig.PodEnvironmentConfigMap.Name == "" {
-		return configMapPodEnvVarsList, nil
+		return nil, nil
```

Comment:
Why ? nil slice and empty slice are different things. In that case it does not seem to have any effect, but in general case might change some behaviour